src/io/sampling.jl

```diff
@@ -350,6 +350,53 @@ function fix_factor!(d::Domain; factors...)::Nothing
     end
 end
 
+"""
+    set_factor_bounds!(d::Domain, factor::Symbol, new_bnds::Tuple)
+    set_factor_bounds!(d::Domain; factors...)
+
+Fix bounds of a parameter for sampling to those provided.
+
+Note: Changes are permanent. To reset, either specify the original value(s)
+      or reload the Domain.
+
+# Examples
+```julia
+# Fix `wave_stress` to specified bounds
+set_factor_bounds!(dom, :wave_stress, (0.1,0.2))
+```
+"""
+function set_factor_bounds!(d::Domain, factor::Symbol, new_bnds::Tuple)::Nothing
+    params = DataFrame(d.model)
+
+    # Check new parameter bounds are within old parameter bounds
+    if (new_bnds[1] < params[params.fieldname .== factor, :default_bounds][1][1]) ||
+        (new_bnds[2] > params[params.fieldname .== factor, :default_bounds][1][2])
```

Comment:
Assign bounds to variable for readability:

```
default_lb, default_ub = params[params.fieldname .== factor, :default_bounds][1]
new_lb, new_ub = new_bnds

if (new_lb < default_lb) || (new_ub > default_ub)
    ...
end

# You could go a step further too
out_of_bounds = (new_lb < default_lb) || (new_ub > default_ub)
if out_of_bounds
    ...
end
```

Reply:
If `upr` and `lwr`, I think it's clearer if you just write the entire thing out -> `default_upper`; `default_lower`

I prefer `lb` / `ub` as its an established convention within scientific code for upper/lower bounds but @Zapiano is resistant to conformity and prefers clarity.

Reply:
I'll change this